miden-lib/asm/wallet/basic.masm

```diff
@@ -0,0 +1,30 @@
+use.miden::sat::account
+use.miden::sat::tx
+use.miden::eoa::basic_auth
+
+#! Stack: [ASSET]
+#! Output: [ASSET']
+#!
+#! - ASSET' final asset in the account vault defined as follows:
+#!   - If ASSET is a non-fungible asset, then ASSET' is the same as ASSET.
+#!   - If ASSET is a fungible asset, then ASSET' is the total fungible asset in the account vault
+#!     after ASSET was added to it.
+export.receive_asset
+    exec.account::add_asset
+end
+
+#! Creates a note which sends a single asset to the specified recipient.
+#!
+#! Stack: [ASSET, tag, RECIPIENT]
+#! Output: [ptr]
+#!
+#! - ASSET is the non-fungible asset of interest.
+#! - tag is the tag to be included in the note.
+#! - RECIPIENT is the recipient of the note.
+#!
+#! - ptr is the pointer to the memory address at which the note is stored.
+export.send_asset
+    exec.account::remove_asset
+    exec.tx::create_note
+    exec.basic_auth::auth_tx
```

Comment:
We don't need to do `auth_tx` here. The authentication script should be called as a part of the transaction script rather than a wallet interface.

Reply:
done, I call it now in the tx script in the test and it works. However, the test is not useful yet, waiting for the mock data framework